test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo docker build --force-rm -t pytrac/debian:stable .
+sudo docker run -rm -t -i  -v $PWD:/home/pytrac/data pytrac/debian:stable ./venv/bin/python ./setup.py test
```

Comment:
@mlafeldt a solution might be that linux users start the test.sh via sudo, but I don't like that either.